server/routes/createGoal/createGoalFormValidator.ts

```diff
@@ -0,0 +1,12 @@
+import type { CreateGoalForm } from 'forms'
+import validateGoalReviewDate from '../../validators/goalReviewDateValidator'
+import formatErrors from '../errorFormatter'
+import validateGoalTitle from '../../validators/goalTitleValidator'
+
+export default function validateCreateGoalForm(createGoalForm: CreateGoalForm): Array<Record<string, string>> {
+  const errors: Array<Record<string, string>> = []
+
+  errors.push(...formatErrors('title', validateGoalTitle(createGoalForm.title)))
+  errors.push(...formatErrors('reviewDate-day', validateGoalReviewDate(createGoalForm.reviewDate)))
```

Comment:
Have you chosen `-day` because you have to select one of the fields for the errors to work at the top of the page? Interestingly however, I note that the review date fields don't have a red border around them like the goal title does.

Reply:
That is a good question and a good observation ...
Honestly .... I copied it from what we did in SLM (surprise!!!! 😁 ), but there was some potentially nasty hacky stuff that I deliberately didnt copy that I believe was about styling the whole field set with a red border 

I didn't like the apparent hackyness of it, certainly considering this is the GDS templates, components and styling ... so should just work ...

Let me look at it again and see what I can do / remember / hack together 😁